src/modules/app/containers/App.js

```diff
@@ -10,18 +9,68 @@ import languagesEndpoint from '../../endpoint/endpoints/languages'
 import categoriesEndpoint from '../../endpoint/endpoints/categories'
 import locationEndpoint from '../../endpoint/endpoints/locations'
 import eventsEndpoint from '../../endpoint/endpoints/events'
+import CategoriesPage from 'routes/categories/containers/CategoriesPage'
+import SearchPage from '../../../routes/search/containers/SearchPage'
+import DisclaimerPage from '../../../routes/disclaimer/containers/DisclaimerPage'
+import EventsPage from '../../../routes/events/containers/EventsPage'
+import PdfFetcherPage from '../../../routes/pdf-fetcher/containers/PdfFetcherPage'
+import LandingPage from '../../../routes/landing/containers/LandingPage'
+import MainDisclaimerPage from '../../../routes/main-disclaimer/components/MainDisclaimerPage'
+import RouteConfig from '../RouteConfig'
+import RouterFragment from './RouterFragment'
 
 class App extends React.Component {
+  store
+  routeConfig
+
   componentWillMount () {
-    this._store = createReduxStore(createHistory)
+    const routes = [
+      {
+        path: '/'
```

Comment:
id: LandingPage would be nice I think

Reply:
LandingPage is not used right now for linking. We can add it later if we want to.